app/services/tax_host.rb

```diff
@@ -148,7 +148,15 @@ def non_inherited_ids(v1 = self.selected_ids, v2 = self.inherited_ids)
     # end
     define_method "#{key}s".to_sym do
       # can't use pluck(key), :domain_id is delegated method, not SQL column, performance diff wasn't big
-      hosts.map(&key).uniq.compact
+      ids = []
+      hosts.each do |host|
+        host.host_facets.each do |facet|
```

Comment:
@dLobatog , I will create a compatibility properties declaration in `Facets::Entry`, that will add pass through with deprecation properties in host object. This way we will not need to change tax_host right now.
``` Ruby
#on Host::Managed
def environment_id
   deprecation_warning
   puppet_facet.environment_id if puppet_facet
end
```